plume-models/src/comments.rs

```diff
@@ -157,14 +157,20 @@ impl Comment {
 
     pub fn notify(&self, conn: &Connection) -> Result<()> {
         for author in self.get_post(conn)?.get_authors(conn)? {
-            Notification::insert(
-                conn,
-                NewNotification {
-                    kind: notification_kind::COMMENT.to_string(),
-                    object_id: self.id,
-                    user_id: author.id,
-                },
-            )?;
+            if Mention::list_for_comment(conn, self.id)?
+                .iter()
+                .all(|m| m.get_mentioned(conn).map(|u| u != author).unwrap_or(true))
```

Comment:
Shouln't this be `.any(...)`? Because I'm still getting duplicated notifications.

Reply:
if *all* mentioned users are different from this author, send notification. I think it's correct, and when I tested it, it seemed to work

Reply:
Oh, I found what is wrong I think: the mention was inserted after the comment in the database.